packages/runtime/src/helpers/functions.ts

```diff
@@ -208,6 +225,143 @@ export const setupImageFunction = async ({
   }
 }
 
+const traceRequiredServerFiles = async (publish: string): Promise<string[]> => {
+  const requiredServerFilesPath = join(publish, 'required-server-files.json')
+  const {
+    files,
+    relativeAppDir,
+    config: {
+      experimental: { outputFileTracingRoot },
+    },
+  } = (await readJSON(requiredServerFilesPath)) as {
+    files: string[]
+    relativeAppDir: string
+    config: { experimental: { outputFileTracingRoot: string } }
+  }
```

Comment:
We do have a utility function `getRequiredServerFiles` which should be used here instead

Reply:
Good catch! done in 88f9b3ea41a43f4d16246eadbfa66c03e82b7a3f.